editor/src/clj/editor/field_expression.clj

```diff
@@ -17,7 +17,8 @@
             [editor.math :as math])
   (:import (java.math MathContext)
            (java.text DecimalFormat DecimalFormatSymbols)
-           (java.util Locale)))
+           (java.util Locale)
+           [net.objecthunter.exp4j ExpressionBuilder]))
```

Comment:
Style: There's a mix of brackets here. Many times you can use **Code > Optimize Imports** in IntelliJ to ensure the requires and imports are consistent and also sorted, which helps with merge conflicts between branches that each add imports to the same file. Unfortunately IntelliJ will sometimes not understand that something is being used and will strip out a needed import, and `ExpressionBuilder` seems to be a victim of that particular bug. 😞

I'll let you decide if you want to change the `()`-wrapped imports to `[]`, change your added import to `()`, or do nothing about it, but at least I got to tell you about the **Optimized Imports** feature. It really works most of the time and can be a great way to keep the imports organized! 🙂

Reply:
Yup, I just use `Optimized Imports` features.

Thank you!